agents-docs/quick-reference.md

```
@@ -44,6 +44,33 @@ Run before commit (see `git-workflow` skill for details):
 scripts/validate.sh
 ```
 
+## CLI Surface Lock (ADR-0066)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)